content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -722,6 +722,92 @@ controller is tracking a Job using Pod finalizers by checking if the Job has the
 annotation `batch.kubernetes.io/job-tracking`. You should **not** manually add
 or remove this annotation from Jobs.
 
+### Specifying your Pod failure policy
```

Comment:
Move this before "Job tracking with finalizers", because this feature is more relevant for users, whereas "Job tracking with finalizers" is more of an implementation detail.

Reply:
done